codingTest_김준우/모의고사.js

```diff
@@ -0,0 +1,42 @@
+// 수포자 1 2 3 가 찍는 방식이 정해져 있다.
+// 가장 많이 맞춘 사람을 배열에 넣어 리턴한다
+// 가장 많이 맞춘 사람이 두 명 이상인 경우 오름차순으로 정렬해서 리턴한다.
+// 수포자 1 : 1 2 3 4 5 / 1 2 3 4 5 / 1 2 3 4 5
+// 수포자 2 : 2 1 2 3 2 4 2 5 / 2 1 2 3 2 4 2 5
+// 수포자 3 : 3 3 1 1 2 2 4 4 5 5 / 3 3 1 1 2 2 4 4 5 5
+// 정답지 answers -> 배열
+function solution(answers) {
+    let supo1 = [1, 2, 3, 4, 5];
+    let supo2 = [2, 1, 2, 3, 2, 4, 2, 5];
+    let supo3 = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];
+    let key = [0, 0, 0];
+    let value = [];
+    for(let i =0; i< answers.length; i++) {
```

Comment:
```suggestion
    let correct = [0, 0, 0]; // 혹은 answer 와 같은 명시적으로 한눈에 알아볼 수 있는 변수명
    let value = [];
    for(let i = 0; i < answers.length; i++) {
```
잘 푸셨습니다 띄어쓰기랑 변수명 설정에 대해서 조금만 더 신경써주세요!!